Makefile

```diff
@@ -47,14 +47,29 @@ GO111MODULE = on
 # ====================================================================================
 # Setup Kubernetes tools
 
+UP_VERSION = v0.13.0
+UP_CHANNEL = stable
 -include build/makelib/k8s_tools.mk
 
 # ====================================================================================
 # Setup Images
 
-DOCKER_REGISTRY ?= crossplane
-IMAGES = provider-jet-equinix provider-jet-equinix-controller
--include build/makelib/image.mk
+IMAGES = provider-equinix
+-include build/makelib/imagelight.mk
```

Comment:
The build/ directory in this project is a copy of the relevant files (pattern taken from the provider-jet-template) from https://github.com/upbound/build/commit/e018ec6.

Reply:
We may need to update build files if this is not sufficient.

Reply:
I _think_ this should be sufficient, but updating to latest may be required for Go to work in CI after merge